engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/VariableInstanceManager.java

```diff
@@ -96,4 +97,10 @@ protected void configureQuery(VariableInstanceQueryImpl query) {
     getTenantManager().configureQuery(query);
   }
 
+  @SuppressWarnings("unchecked")
+  public List<VariableInstanceEntity> selectByBatchId(String batchId) {
```

Comment:
To stay consistent here, maybe
```suggestion
  public List<VariableInstanceEntity> findVariableInstancesByBatchId(String batchId) {
```

Reply:
👍